python/taichi/lang/matrix.py

```diff
@@ -156,6 +140,8 @@ def element_wise_binary(self, foo, other):
                 ret.entries[i] = foo(self.entries[i], other)
         return ret
 
+    element_wise_inplace_binary = element_wise_binary
```

Comment:
I don't think this is inplace, but I guess it's pretty easy to make one. E.g.

```py
def element_wise_inplace_binary(self, foo, other):
  self.assign(self.element_wise_binary(foo, other))
```

Reply:
Sorry about the confusing name... maybe it's actually `element_wise_atomic_binary` :)